model/release.go

```diff
@@ -26,6 +26,8 @@ type Release struct {
 	Links                     []Link             `json:"links"`
 	DateChanges               []DateChange       `json:"date_changes"`
 	Description               ReleaseDescription `json:"description"`
+	// TODO Provisional entry for modelling history
+	// ReleaseHistory            []Link         `json:"release_history"`
 }
 
```

Comment:
Public structs and methods have a go doc that states what they are for
```suggestion
// DateChange represents a single date change
```